modules/oc-mirror-migration-differences.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * disconnected/mirroring/oc-mirror-migration-v1-to-v2.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oc-mirror-migration-differences_{context}"]
+= Changes from oc-mirror plugin v1 to v2
+
+Before migrating from oc-mirror plugin v1 to v2, see the following differences between oc-mirror plugin v1 and v2:
+
+* Explicit version selection: Users must explicitly specify `--v2` when using `oc-mirror`. If no version is specified, v1 is executed by default. This behavior is expected to change in future releases, where `--v2` will be the default.
+
+* Updated commands: Commands for mirroring workflows have changed to align with oc-mirror plugin v2's new workflow.
+
+** For mirror-to-disk, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml file://<directory_name> --v2
+----
+
+** For disk-to-mirror, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml --from file://<directory_name> docker://<remote_registry> --v2
+----
+
+** For mirror-to-mirror, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml --workspace file://<directory_name> docker://<remote_registry> --v2
+----
++
+[NOTE]
+====
+`--workspace` is now required for mirror-to-mirror operation.
+====
+
+. API version update: The `ImageSetConfiguration` API version changes from `v1alpha2` (v1) to `v2alpha1` (v2). You must manually update the configuration files before migration.
+
+. Configuration changes: 
```

Comment:
The rest of the list items in this section starting from here and going to line 60 still need to be changed to unordered bullets (`.` changed to `*`). Right now there is a mix and that definitely must be fixed before merging:
```suggestion
* API version update: The `ImageSetConfiguration` API version changes from `v1alpha2` (v1) to `v2alpha1` (v2). You must manually update the configuration files before migration.

* Configuration changes: 
```